app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::ReviewsController < ApplicationController
+  def show
+    render json: Review.find(params[:id])
+  end 
```

Comment:
Is this controller and controller action being used at all? If not we can remove it

Reply:
Removed